### PR TITLE
Fix wording >  introduction_to_grid.md

### PR DESCRIPTION
**Pull request by elayachihamrit1**
## Because

Improving the wording in a sentence to provide a more positive and encouraging tone.

## This PR

Adjusting the sentence "Remember how much fun you had solving the card layout in this exercise? We know that was a frustrating one, but it’s part of the point." to "Remember the card layout exercise? It challenged your understanding of Flexbox, but that's how you learn and grow as a developer."

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [ ] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [ ] The `Because` section summarizes the reason for this PR
-   [ ] The `This PR` section has a bullet point list describing the changes in this PR
-   [ ] If this PR addresses an open issue, it is linked in the `Issue` section
-   [ ] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [ ] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Review by KevinMulhern — CHANGES_REQUESTED** [review]
Thanks @elayachihamrit1, I like the sentiment of this change. Just a minor suggestion

File: intermediate_html_css/grid/introduction_to_grid.md [hunk 1]
```diff
@@ -43,7 +43,9 @@ Remember how much fun you had solving the card layout in this exercise?
 
 [![flex-exercise-desired-outcome.png](https://i.postimg.cc/vZ81HMkB/flex-exercise-desired-outcome.png)](https://github.com/TheOdinProject/css-exercises/tree/main/flex/07-flex-layout-2)
 
-We know that was a frustrating one, but it's part of the point. While Flexbox allows you to build a layout of rows and columns together, it isn’t always easy.
+It challenged your understanding of Flexbox, but that's how you learn and grow as a developer."
```
Comment (KevinMulhern) [inline intermediate_html_css/grid/introduction_to_grid.md:46]: Nit: This reads a little better to me: 

```suggestion
That was a challenging one! but that's how you learn and grow as a developer."
```

**Comment by elayachihamrit1** [comment]
Thanks ❤️

File: intermediate_html_css/grid/introduction_to_grid.md (hunk 1 above)
Comment (elayachihamrit1) [reply intermediate_html_css/grid/introduction_to_grid.md]: solved

**Review by elayachihamrit1 — COMMENTED** [review]
solved

**Pull request closed without merging**